### PR TITLE
`ocean interface` - wait for transaction confirmation

### LINKED ISSUE #306: `ocean interface` - wait for the transaction to be confirmed
#### What would you like to be added:
Once a transaction is broadcasted, wait for the transaction to be confirmed and display. Endpoint for validation - https://github.com/DeFiCh/whale/pull/244

### PR DESCRIPTION
**Pull request by thedoublejay**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Ability to wait for transaction to be confirmed. Also, adds fetching of balance after a transaction. 
#### Which issue(s) does this PR fixes?:

Fixes #306

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -28,38 +28,44 @@ import '@testing-library/cypress/add-commands'
...
-			sendDFITokentoWallet (): Chainable<Element>
-	
-			/**
-			 * @description Sends token to wallet. Accepts a list of token symbols to be sent.
-			 * @param {string[]} tokens to be sent
-			 * @example cy.sendTokenToWallet(['BTC', 'ETH']).wait(4000)
-			 */
-			sendTokenToWallet (tokens: string[]): Chainable<Element>
-		}
+      /**
```
Comment (ivan-zynesis) [inline cypress/support/commands.ts:31]: Seems like the changes here is from `tab` to 2 `space` indentation, can you revert it?

File: app/components/OceanInterface/OceanInterface.tsx [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 import { useWhaleApiClient } from '../../contexts/WhaleContext'
+import { fetchTokens } from '../../hooks/wallet/TokensAPI'
 import { RootState } from '../../store'
 import { firstTransactionSelector, ocean, OceanTransaction } from '../../store/ocean'
 import { tailwind } from '../../tailwind'
 import { translate } from '../../translations'
 
 const MAX_AUTO_RETRY = 1
+const MAX_TIMEOUT = 300000
+const TIMEOUT_INTERVAL = 30000
```
Comment (ivan-zynesis) [inline app/components/OceanInterface/OceanInterface.tsx:20]: I think we need extra config

- TIMEOUT: give up
- INTERVAL_1ST: getNetwork === playground ? 3000++ : 30000++
- INTERVAL_REST: 1000 (block should be ready anytime soon)

**Review by ivan-zynesis — CHANGES_REQUESTED** [review]

File: cypress/support/commands.ts (hunk 1 above)
Comment (thedoublejay) [reply cypress/support/commands.ts:31]: oh yeah good point, let me check

File: app/components/OceanInterface/OceanInterface.tsx (hunk 2 above)
Comment (thedoublejay) [reply app/components/OceanInterface/OceanInterface.tsx]: Updated this @ivan-zynesis, what I did is to call an initial timeout. Then the succeeding interval, I put 5 secs so it won't spam API.

**Review by fuxingloh — APPROVED** [review]

**Review by ivan-zynesis — APPROVED** [review]

File: app/components/OceanInterface/OceanInterface.tsx [hunk 3]
```diff
@@ -50,6 +87,7 @@ export function OceanInterface (): JSX.Element | null {
   const { height, err: e } = useSelector((state: RootState) => state.ocean)
   const transaction = useSelector((state: RootState) => firstTransactionSelector(state.ocean))
   const slideAnim = useRef(new Animated.Value(0)).current
+  const address = useSelector((state: RootState) => state.wallet.address)
```
[the remark this answers is not in the document]
Comment (thedoublejay) [reply app/components/OceanInterface/OceanInterface.tsx:90]: I'll create a ticket for it

**Pull request merged**